contrib/debian/fwupd-tests.install

```diff
@@ -4,5 +4,6 @@
 #https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=872458
 usr/share/installed-tests/*
 usr/lib/*/fwupd-plugins-3/libfu_plugin_test.so
+usr/lib/*/fwupd-plugins-3/libfu_plugin_test_invalid_build_hash.so
```

Comment:
I'm fine with just making above line test*.so to cover this in the future if we add more

Reply:
I renamed it `libfu_plugin_invalid.so` to keep the daemon blacklist sort.